sbi/neural_nets/density_estimators/zuko_flow.py

```diff
@@ -101,6 +101,8 @@ def log_prob(self, input: Tensor, condition: Tensor) -> Tensor:
         Args:
             input: Inputs to evaluate the log probability on. Of shape
                 `(sample_dim, batch_dim, *event_shape)`.
+            # TODO: the docstring is not correct here. in the code it seems we
+            do not have a sample_dim for the condition.
```

Comment:
we do not have a `sample_dim` for the condition, but we have a `sample_dim` for the `input`. I think the docstring is correct